src/prop/sat_solver.h

```diff
@@ -170,8 +170,23 @@ class CDCLTSatSolver : public SatSolver
    */
   virtual std::vector<Node> getOrderHeap() const = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is PROOF.
+   * Returns a complete proof computed by this SAT solver.
+   */
   virtual std::shared_ptr<ProofNode> getProof() = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is SKETCH.
```

Comment:
Ditto. Also, what is a proof sketch vs a proof? I assume this does not return the same kind of proof as `getProof()`, so maybe this should say `Get proof sketch, i.e., ...... This is used if ...`?